ACMAS/Docker deployment instructions.md

```diff
@@ -0,0 +1,20 @@
+# Deployment instructions for ACMAS @ RCOS v2 (Django, Docker, Postgres, Gunicorn, Nginx)
+## Production server
+_Note: Running migrations and serving static files are now handled in entrypoint.prod.sh. and entrypoint.sh._
+
+[Docker](https://docs.docker.com/get-docker/) is a pre-requisite installation to run the deployment server.
+If you do not have Docker installed, please do so before continuing to the remainder of the instructions.
+### Build images and spin up production containers
+`docker-compose -f docker-compose.prod.yml up -d --build`
```

Comment:
Not sure what your intended audience is with this, but speaking from someone who has docker installed on my computer but no other background in docker, I have no idea what this means. If this is supposed to be for someone who knows about docker and navigating it that's fine. However, for this, I don't know where to run this, assuming it should be put in a terminal, or where any of that goes. So a little more clarity about that would be good, but again, if it is good for people who know docker and it doesn't matter about inexperienced docker users, then you are fine.